npm-debug.log

```diff
@@ -0,0 +1,27 @@
+0 info it worked if it ends with ok
```

Comment:
Remove this file. You should add it to your gitignore file so it doesn't get checked in.